scripts/Iris.json

```diff
@@ -0,0 +1,88 @@
+{
+    "citation": "R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.
+
+",
```

Comment:
So about that.  On my system, I had a copy which showed up and I mainly focused on `version.txt`.